src/Components/CreateImageWizard/steps/Packages/Packages.tsx

```diff
@@ -940,6 +968,39 @@ const Packages = () => {
   const isGroupExpanded = (group: GroupWithRepositoryInfo['name']) =>
     expandedGroups.includes(group);
 
+  const isPackageSelected = (pkg: IBPackageWithRepositoryInfo) => {
+    let isSelected = false;
+
+    if (!pkg.sources || pkg.sources[0].type === 'package') {
+      isSelected = packages.some((p) => p.name === pkg.name);
+    }
+
+    if (pkg.sources && pkg.sources[0] && pkg.sources[0].type === 'module') {
+      isSelected =
+        packages.some((p) => p.name === pkg.name) &&
+        modules.some(
+          (p) =>
+            pkg.sources && pkg.sources[0] && p.stream === pkg.sources[0].stream
+        );
```

Comment:
I think this could do with a simple comment on what exactly it's expecting from the package with module source.

If I understand it correctly, it only says the package is selected if its module stream matches the one of the selected modules' stream? 

Also, what happens for old blueprints which have packages selected which do belong to a module stream, but where we didn't save the module information? They will still be in the packages store data, but not marked as selected?

Reply:
Added the comment.

Also great question, tried to create a blueprint with `nginx` packages and open it in edit mode on the branch - the packages are selected, but they don't have a stream associated (the column might need some value).
![image](https://github.com/user-attachments/assets/a3c5fd1f-9453-4b50-b7f0-880f1635ff5a)